nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2129,6 +2128,37 @@ java.arg.16=-Dsun.security.krb5.debug=true
 This will cause the debug output to be written to the NiFi Bootstrap log file. By default, this is located at _$NIFI_HOME/logs/nifi-bootstrap.log_.
 This output can be rather verbose but provides extremely valuable information for troubleshooting Kerberos failures.
 
+[[zk_tls]]
+=== Securing ZooKeeper with HTTPS
```

Comment:
It's not actually HTTPS, so I think we should be more generic and say TLS or SSL.